runtime/compiler/x/codegen/J9UnresolvedDataSnippet.cpp

```diff
@@ -154,7 +154,7 @@ J9::X86::UnresolvedDataSnippet::emitResolveHelperCall(uint8_t *cursor)
    // Call to the glue routine
    //
    const intptrj_t rip = (intptrj_t)(cursor+5);
-   if ((cg()->fej9()->helpersNeedRelocation() && TR::Compiler->target.is64Bit()) ||
+   if ((cg()->fej9()->needRelocationsForHelpers() && TR::Compiler->target.is64Bit()) ||
```

Comment:
Since you have the new API you could have called `cg()->needRelocationsForHelpers()` directly, but there is nothing wrong with the current form.

Reply:
Thanks for catching that! I've fixed this in: 309fedd